modular_skyrat/modules/cum/chemistry/reagents/cum_reagent.dm

```diff
@@ -0,0 +1,27 @@
+/datum/reagent/cum
+	name = "Cum"
+	description = "A fluid containing sperm that is secretated by the sexual organs of most species."
+	color = "#fff9f1"
+	taste_description = "musk"
+	glass_icon_state = "glass_white"
+	glass_name = "glass of cum"
+	glass_desc = "O-oh, my...~"
+	shot_glass_icon_state = "shotglasswhite"
+
+/datum/reagent/cum/expose_mob(mob/living/exposed_mob, methods=TOUCH, reac_volume)
+	. = ..()
+	if(!(methods & (INGEST|INJECT|PATCH))) //it is intentional that you can still cumshot people with vapor ;)
+	{
```

Comment:
Purpose of these?

Reply:
i noticed that for reason when you ingest/inject the reproductive agent, it would always count as also having touched it, which i didnt want to happen. so, thats why i threw that check in

Reply:
I more mean the {}

Reply:
oh. to be honest, i kinda thought you had to have the brackets at first, although i later realized you just had to indent it. it would probably take me like 5 seconds to get it out if you wanted

Reply:
Yes please, remove the brackets